src/System.CommandLine/CommandLineConfiguration.cs

```diff
@@ -22,15 +22,14 @@ public class CommandLineConfiguration
         /// <summary>
         /// Initializes a new instance of the CommandLineConfiguration class.
         /// </summary>
-        /// <param name="command">The symbol to parse.</param>
+        /// <param name="command">The root command for the parser.</param>
```

Comment:
just a question: should it be always root command? or can it be also subcommand? (if subcommand is reparsed)?

Reply:
It can be either `RootCommand` or `Command`. You can build a parser just for subtree of your command hierarchy.